common/src/stack/command/stack/commands/__init__.py

```diff
@@ -1893,49 +1893,49 @@ def endOutput(self, header=[], padChar='-', trimOwner=False, trimHeader=False):
 				for i in range(0, rows):
 					header.append('col-%d' % i)
 
-			list = []
+			output_lines = []
 			for line in self.output:
-				dict = {}
+				dictionary = {}
 				for i in range(0, len(header)):
 					if header[i]:
 						key = header[i]
 						val = line[i]
 
-						if key in dict:
-							if not isinstance(dict[key], list):
-								dict[key] = [dict[key]]
-							dict[key].append(val)
+						if key in dictionary:
+							if not isinstance(dictionary[key], output_lines):
```

Comment:
Not your code, but I'm not sure what the hell this line of code is checking for. `isinstance` should be taking a class  or a type, not a list of strings. I'm not sure this code does what the original author thought it did?

Reply:
Ah, I see it *was* originally checking for a list. It should still do that, or use `defaultdict` to simplify the whole mess, as Bill suggests.